utils/build-script-impl

```diff
@@ -100,7 +100,7 @@ KNOWN_SETTINGS=(
     installable-package         ""               "the path to the archive of the installation directory"
     test-installable-package    ""               "whether to run post-packaging tests on the produced package"
     reconfigure                 ""               "force a CMake configuration run even if CMakeCache.txt already exists"
-    swift-sdks                  ""               "build target binaries only for specified SDKs (semicolon-separated list)"
+    build-stdlib-deployment-targets "all"        "build target binaries only for specified SDKs (semicolon-separated list)"
```

Comment:
It feels really strange to call it `build-*`, since it is about configuration.  Or did the semantics change?